docs/source/conf.py

```diff
@@ -222,47 +226,17 @@ def setup(app):
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "pydata_sphinx_theme"
+html_theme = "pymc_sphinx_theme"
 
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 
 html_theme_options = {
-    "icon_links": [
```

Comment:
@twiecki answering https://github.com/pymc-devs/pymc-examples/pull/465#discussion_r1021708138 here. If you look at the preview you will see that the logos are still there. What is being removed is the triplication of this info. Now this is part of the pymc-sphinx-theme which inherits from pydata-sphinx-theme and sets up most of the common elements between al 3 main websittes (and others like pymc-experimental or pymc-bart)

Reply:
Amazing!